__tests__/app.test.js

```diff
@@ -46,3 +35,37 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: returns an object containing information on the requested article", () => {
+    return request(app)
+      .get("/api/articles/7")
+      .then(({ body }) => {
+        expect(body.article).toEqual({
```

Comment:
You could use objectContaining matcher instead of toEqual, though that in future you can add properties onto article objects without changing this test, but this will work for now!